src/OpenTok/Broadcast.php

```diff
@@ -34,6 +34,10 @@
 *
 * @property boolean $isStopped
 * Whether the broadcast is stopped (true) or in progress (false).
+*
+* @property string $streamMode
```

Comment:
I think we should create a StreamMode class with constants. (See the ArchiveMode class.)